lib/cocoapods-search/command/search.rb

```diff
@@ -82,7 +84,19 @@ def local_search
           sets.reject! { |set| !set.specification.available_platforms.map(&:name).include?(platform) }
         end
 
-        sets.reverse_each do |set|
+        if @use_pager
+          IO.popen((ENV['PAGER'] || 'less -R'), 'w') do |io|
```

Comment:
Hmm, then I can define a `with_pager` method in UI which takes a block parameter. It will handle `popen` logic internally, and call passed block inside popen block? Do I get you right?